scaletest/createworkspaces/run_test.go

```diff
@@ -257,24 +262,35 @@ func Test_Runner(t *testing.T) {
 			close(done)
 		}()
 
+		// Wait for the workspace build job to be picked up.
 		require.Eventually(t, func() bool {
 			workspaces, err := client.Workspaces(ctx, codersdk.WorkspaceFilter{})
 			if err != nil {
 				return false
 			}
+			if len(workspaces.Workspaces) == 0 {
+				return false
+			}
 
-			return len(workspaces.Workspaces) > 0
-		}, testutil.WaitShort, testutil.IntervalFast)
+			ws := workspaces.Workspaces[0]
+			t.Logf("checking build: %s | %s", ws.LatestBuild.Transition, ws.LatestBuild.Job.Status)
+			// There should be only one build at present.
+			if ws.LatestBuild.Transition != codersdk.WorkspaceTransitionStart {
+				t.Fatalf("expected build transition %s, got %s", codersdk.WorkspaceTransitionStart, ws.LatestBuild.Transition)
```

Comment:
can't call `t.Fatalf` in an Eventually condition, because it is run on a separate go routine and `t.Fatalf` is only allowed on the main test goroutine.

Reply:
oops